### DISCUSSION TITLE
Add fields to airbyte_raw_tables

### DISCUSSION THREAD

**Discussion by gerbetta2992** [airbytehq/airbyte#29129] [Connector Questions]
Hi, my name is Germán, I work for fortisoft, a company in charge of offering cloud solutions and data management through connectors. Right now we are developing a custom connector with airbyte and we need to add a field to an endpoint taken from another endpoint. When creating the connector with the changes made in the code, we see that the record brings the data just as we added it. The problem is that when the _airbyte_raw_endpoint table is created, we did not find the data added to the _airbyte_data column of that table. Is there a way to do this without having to perform normalization? 

The following image is a print of the record showing the data that we want to add, I also show an image of the data added to the list of fields of the source. Having only that field selected the airbyte_data column comes as an empty dictionary

![image](https://github.com/airbytehq/airbyte/assets/68885439/6d28e7ee-2f9e-40db-85bb-5b75fe5d8f65)
![image](https://github.com/airbytehq/airbyte/assets/68885439/8cf70f6f-9a8b-4b96-9168-b07efebc3ec8)

**Comment by marcosmarxm** [comment]
I closed the discussion because it was inactive. Please feel free to open a new one, our team will provide some feedback about your question.
